e2e/cypress/integration/system_console/plugin_upload_spec.js

```diff
@@ -1,106 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-// ***************************************************************
-// - [#] indicates a test step (e.g. # Go to a page)
-// - [*] indicates an assertion (e.g. * Check the title)
-// - Use element ID when selecting an element. Create one if none.
-// ***************************************************************
-
-// Stage: @prod
-// Group: @not_cloud @system_console @plugin
-
-/**
- * Note : This test requires draw plugin tar file under fixtures folder.
- * Download from : https://integrations.mattermost.com/draw-plugin/
- * Copy to : ./e2e/cypress/fixtures/com.mattermost.draw-plugin.tar.gz
- */
-
-import * as TIMEOUTS from '../../fixtures/timeouts';
-
-describe('Draw Plugin - Upload', () => {
```

Comment:
Deleted since this is similar to [MM-T2400](https://github.com/mattermost/mattermost-webapp/pull/7981/files#diff-777de0aa14cc7eb470b5f4c5be47382726d1c3cd698565505b3a65144c9f28beR32).